{lyn15598} Explaining the name field matching ability for ACS configuration (#2018)
o3de/o3de.org: content/docs/user-guide/assets/asset-processor/asset-cache-server.md

This ACS block allows users to configure the types of source assets that should be cached in the remote folder. The block is placed in the JSON path `/AssetProcessor/Settings/Server` object. The title must start with the prefix `"ACS "` to designate the object as a configuration block. The next part is either `"glob"` or `"pattern"` followed by the correct text for a wild card pattern or a regular expression; these are used to tag source assets that need to be cached.

The ``name`` field is used for both the title of the configuration block in the GUI tool, but it is also possible to match all an asset pattern by setting the ``name`` to a name of an asset builder. For example, a configuration block can set the name to "Atom Image Builder" so that all the processed images will be cached.

{{< note >}}
The block should only be set to `"glob"` or `"pattern"`, not both.
{{< /note >}}

The `"checkServer"` Boolean flag is used to enable the block. The default value for `"checkServer"` is false, so to enable the ACS block the Boolean flag needs to be set to true.

> An example this JSON ACS block will cache all product assets built by the "XmlBuilderWorker" builder:

Reviewer: While its true it does capture all the XML files produced, it may be good to expand the description somewhat for example:

```suggestion
This example JSON ACS block cache all product assets built by the "XmlBuilderWorker" builder. In this case its all the processed `.vegdescriptorlist` xml files:
```

Author: Updated

```json
"ACS XmlBuilderWorker":
{

>    "name": "XmlBuilderWorker",

Reviewer: nit: Above we have "Atom Image Builder" and here "XMLBuilderWorker". Does white space matter in name matching? Maybe good to callout either way.

Author: White space does matter. 

> For example, a configuration block can set the name to "Atom Image Builder" so that all the processed images will be cached.

I can add a note that the builder name must match exactly (spaces and case)

Author: Updated

The `"checkServer"` Boolean flag is used to enable the block. The default value for `"checkServer"` is false, so to enable the ACS block the Boolean flag needs to be set to true.

> The ``name`` field is used for both the title of the configuration block in the GUI tool, but it is also possible to match all an asset pattern by setting the ``name`` to a name of an asset builder. For example, a configuration block can set the name to "Atom Image Builder" so that all the processed images will be cached.

Reviewer 2: I'm having difficulty comprehending this sentence due to the word "both". Is this suggestion still correct?

```suggestion
The ``name`` field is used for the title of the configuration block in the GUI tool, but it can also match all assets of an asset pattern by setting it to the name of an asset builder. For example, a configuration block can set the name to "Atom Image Builder" so that all the processed images are cached.
```

> {{< note >}}
> To use the ``name`` of a builder to cahce its product files it must match exactly includings white space and letter case.
> {{< /note >}}
> 
> This example JSON ACS block will cache all product assets built by the "XmlBuilderWorker" builder:

Reviewer 2: minor suggestion

```suggestion
{{< note >}}
To use the ``name`` of a builder to cache, its product files it must match exactly, including white space and letter case.
{{< /note >}}

This example JSON ACS block caches all product assets built by the "XmlBuilderWorker" builder:
```
